token/services/db/sql/postgres/driver.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package postgres
+
+import (
```

Comment:
I split the postgres driver from the sqlite driver:
* because they have different configs (although they overlap significantly). We shouldn't create configs that contain properties of both solutions and use them partially, because it is not clear what is needed/used and what not. Plus, this is more extensible if we add more db drivers.
* to flatten the config in the next steps